packages/shared/providers.yaml

```diff
@@ -429,6 +429,34 @@ builder-io-public:
         query:
             apiKey: ${apiKey}
     docs: https://docs.nango.dev/integrations/all/builder-io
+cal-com-v1:
+    categories:
+        - productivity
+    auth_mode: API_KEY
+    proxy:
+        base_url: https://api.cal.com/v1
+        query:
+            apiKey: ${apiKey}
+        retry:
+            at: 'X-RateLimit-Reset'
+        verification:
+            method: GET
+            endpoint: /me
+    docs: https://docs.nango.dev/integrations/all/cal-com
+cal-com-v2:
```

Comment:
feels like we are going to hit more and more of those, I wish we had a better way to version providers

Reply:
I agree 💯